dexbot/strategies/staggered_orders.py

```diff
@@ -258,6 +258,8 @@ def allocate_base_asset(self, base_balance, *args, **kwargs):
             else:
                 # Cancel highest buy order
                 self.cancel(self.buy_orders[0])
+                self.place_higher_buy_order(self.buy_orders[0]) # Shouldn't [0] be the new "highest"?
```

Comment:
The if-else checks if there are orders at all, so [0] is automatically the highest, since it is the only order.